src/hardware/developing-an-iox/protocol.md

```diff
@@ -138,6 +138,7 @@ Data sent from the GO device to the addressed IOX.
 ### RX Data (0x0C)
 
 Data sent from an IOX to the GO device. The GO will reply with an ACK.
+The communication between an IOX and GO device starts with a 0x25 message - [IOX Request/Status (0x25)](#iox-requeststatus-0x25) indicating the beginning of the 0x0C message, and also ends with one.
```

Comment:
How about `The 0x0C message series start with a [0x25 message](#iox-requeststatus-0x25), and also ends with one.`? Could you also ensure that the hyperlink works? Will need you to use the [Local testing approach](https://docs.google.com/document/d/1G0KD3P8RpTemX_qjlvdMzzO0J4yEwNAk05VOEyjbvpA/edit#heading=h.vbqbgxdf5572)